gef.py

```diff
@@ -3835,49 +3845,49 @@ def gef_convenience(value):
     return var_name
 
 
-def parse_string_range(s):
+def parse_string_range(s: str) -> Iterator[int]:
     """Parses an address range (e.g. 0x400000-0x401000)"""
     addrs = s.split("-")
     return map(lambda x: int(x, 16), addrs)
```

Comment:
We should turn this into a list comprehension and simplify the type to `List[int]` or even `Tuple[int, int]`

Reply:
added this thought to the linked project 11